app/graphql/feature-flags-context.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, createContext, useContext, useEffect } from "react"
+import remoteConfigInstance from "@react-native-firebase/remote-config"
+import { useAppConfig } from "@app/hooks"
+
+type FeatureFlags = {
+  deviceAccountEnabled: boolean
+}
+
+type RemoteConfig = {
```

Comment:
we could make this consistent by using only one type.
```suggestion
type RemoteConfig = FeatureFlags;
```
or just extend the `FeatureFlags` incase we want to add more properties to the RemoteConfig